src/client/components/MealList/MealList.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from "react";
+import Meal from "../Meal/Meal";
+import { MealContext } from "../Context/MealContext";
+import "../Meal/meal.css";
+const MealList = () => {
```

Comment:
I later noticed that it was a typo in the instructions 🤦🏽 
Sorry for that. 
But good to keep in mind and maybe change in the next coding round, so that the names are more accurate 🙏🏽

Reply:
Thanks and sure will keep track 🙏🏾.